packages/mediacenter/kodi/scripts/service-addon-wrapper

```diff
@@ -64,3 +64,48 @@ if [ -f "${SERVICE_FILE}" ] ; then
       ;;
   esac
 fi
+
+if [ ! -d /storage/.cache/kernel-overlays ] ; then
```

Comment:
I think that "if [ -d ... ]" "mkdir -p /storage/.cache/kernel-overlays" block could maybe be dropped as the directory should have been created before by the kernel-overlay systemd service.

OTOH it doesn't hurt keeping it as it makes the wrapper config stuff self-contained.

I'll think about it.